package.json

```
@@ -15,7 +15,7 @@
   },
   "homepage": "https://github.com/rsksmart/bridge-transaction-parser#readme",
   "dependencies": {
-    "@rsksmart/rsk-precompiled-abis-lovell700": "git+https://github.com/rsksmart/precompiled-abis#master"
+    "@rsksmart/rsk-precompiled-abis": "npm:@rsksmart/rsk-precompiled-abis@^6.0.0-ARROWHEAD"
```

```suggestion
   "@rsksmart/rsk-precompiled-abis": "^6.0.0-ARROWHEAD",
```
Not sure it changes anything, just the way I'm used to seeing it

---

Done.